diff --git a/providers/email.d.ts b/providers/email.d.ts
index da621e5a8ccbe93639b1f55eb1aff23d26b508cd..2c7557b703dbbeb7affa82f2dd81f94562491410 100644
--- a/providers/email.d.ts
+++ b/providers/email.d.ts
@@ -69,7 +69,8 @@ export interface EmailConfig extends CommonProviderOptions {
      */
     normalizeIdentifier?: (identifier: string) => string;
 }
+export type EmailUserConfig = Partial<Omit<EmailConfig, "options">>;
 export type EmailProviderType = "email";
 /** TODO: */
-export default function Email(config: EmailConfig): EmailConfig;
+export default function Email(config: EmailUserConfig): EmailConfig;
 //# sourceMappingURL=email.d.ts.map
\ No newline at end of file
diff --git a/src/providers/email.ts b/src/providers/email.ts
index 6ee21568096385c6a79580e9fc00ab261aa2584e..d03725ebbe50a59ac20dfed58c9f30414a6d7d9f 100644
--- a/src/providers/email.ts
+++ b/src/providers/email.ts
@@ -77,12 +77,14 @@ export interface EmailConfig extends CommonProviderOptions {
   normalizeIdentifier?: (identifier: string) => string
 }
 
+export type EmailUserConfig = Partial<Omit<EmailConfig, "options">>
+
 // TODO: Rename to Token provider
 // when started working on https://github.com/nextauthjs/next-auth/discussions/1465
 export type EmailProviderType = "email"
 
 /** TODO: */
-export default function Email(config: EmailConfig): EmailConfig {
+export default function Email(config: EmailUserConfig): EmailConfig {
   return {
     id: "email",
     type: "email",